docker/develop/Dockerfile

```diff
@@ -118,14 +149,21 @@ RUN git clone https://github.com/martinmoene/optional-lite /tmp/optional-lite; \
     rm -rf /tmp/optional-lite
 
 # install libpq
-RUN git clone https://git.postgresql.org/git/postgresql.git /tmp/postgresql; \
-    (cd /tmp/postgresql ; git checkout 029386ccbddd0a33d481b94e511f5219b03e6636); \
-    (cd /tmp/postgresql ; /tmp/postgresql/configure --without-readline --prefix=/usr/local); \
-    make -j${PARALLELISM} -C /tmp/postgresql/src/bin/pg_config; \
-    make -j${PARALLELISM} -C /tmp/postgresql/src/interfaces/libpq; \
-    make -C /tmp/postgresql/src/bin/pg_config install; \
-    make -C /tmp/postgresql/src/interfaces/libpq install; \
-    make -C /tmp/postgresql/src/include install; \
+RUN git clone --progress https://git.postgresql.org/git/postgresql.git /tmp/postgresql; \
+    cd /tmp/postgresql; \
+    git checkout 029386ccbddd0a33d481b94e511f5219b03e6636; \
+    ./configure --without-readline --prefix=/usr/local; \
+    # build
+    make -j${PARALLELISM} -C src/bin/pg_config; \
+    make -j${PARALLELISM} -C src/interfaces/libpq; \
+    make -j${PARALLELISM} -C src/backend/utils fmgroids.h; \
+    cp src/backend/utils/fmgroids.h src/include/utils/fmgroids.h; \
+    # install
+    make -C src/bin/pg_config install; \
+    make -C src/interfaces/libpq install; \
+    make -C src/include install; \
```

Comment:
and with https://github.com/hyperledger/iroha/pull/1064/files#diff-6f13a3dbd7d676a11d65a45353f73feaR160 builds works as expected